qmi/instruments/tenma/psu_72.py

```diff
@@ -0,0 +1,490 @@
+        return {
+            "Ch1Mode": "C.V" if ch1mode else "C.C",
+            "Ch2Mode": "C.V" if ch2mode else "C.C",
+            "Tracking": tracking,
+            "OutputEnabled": bool(output_enabled),
+        }
+
+    @rpc_method
+    def enable_output(self, output: bool) -> None:
+        command = f"OUT{int(output)}"
+        self._enable_output(output, command)
+
+
+class Tenma72_2535(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 3
+
+
+class Tenma72_2540(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 5
+
+
+class Tenma72_2545(Tenma72_2550):
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 2
+
+
+class Tenma72_2925(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 5
+
+
+class Tenma72_2930(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 10
+
+
+class Tenma72_2935(Tenma72_2550):
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 5
+
+
+class Tenma72_2940(Tenma72_2550):
+    DEFAULT_BAUDRATE = 9600
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 5
+
+
+class Tenma72_10480(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 3
+
+
+class Tenma72_13350(Tenma72_Base):
+    """Instrument driver for the Tenma 72-13350. The driver is tested with this model, but manual is
+    also for the model 72-13360.
+
+    This driver can be used with (USB-to-)serial and UDP communications.
+
+    Note that this model has extra functionalities related to the IP LAN Communication Protocol.
+    Also note that this model does not work with channel number input, like 2550 model-based PSUs.
+    """
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 30
+    DEFAULT_BAUDRATE = 115200
+
+    def __init__(self, context: QMI_Context, name: str, transport: str) -> None:
+        super().__init__(context, name, transport)
+        self.serial_eol = "\n"
+
+    def _send(self, cmd: str) -> None:
+        cmd = cmd + self.serial_eol
+        return super(Tenma72_13350, self)._send(cmd)
+
+    @rpc_method
+    def get_status(self) -> Dict[str, Any]:
+        """Get the power supply status as a dictionary of status values.
+
+        Dictionary composition is:
+            "ChannelMode ": "C.V" | "C.C",
+            "OutputEnabled": True | False,
+            "V/C priority ": "Current priority" | "Voltage priority",
+            "Beep ": True | False,
+            "Lock ": True | False,
+
+        Returns: 
+            Dictionary of status values.
+        """
+        self._send("STATUS?")
+        statusBytes = self._transport.read(2, 2)  # Read response byte
+        # 72-13350 sends two bytes back, the second being '\n'
+        status = statusBytes[0]
+
+        ch1mode = (status & 0x01)
+        output_enabled = bool(status & 0x02)
+        current_priority = (status & 0x04)
+        beep = bool(status & 0x10)
+        lock = bool(status & 0x20)
+
+        return {
+            "ChannelMode": "C.V" if ch1mode else "C.C",
+            "OutputEnabled": output_enabled,
+            "V/C priority": "Current priority" if current_priority else "Voltage priority",
```

Comment:
this can also be an enum. Makes comparisons easier for the layers above QMI

Reply:
Enum just for two values does not make much of a difference. And I think the client wants to use the string as output so I would prefer to leave it.